src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1963,24 +2120,23 @@ private void Submit()
 
             var contentFiles = BuildPackage();
 
+            //if buildPackage() returns no files then the package
+            //is empty so we should return
+            if (contentFiles == null || contentFiles.Count() < 1) return;
```

Comment:
Use `Any()` for better performance: 
`if (contentFiles == null || !contentFiles.Any()) return;`

Reply:
I think Visual Studio was suggesting to me that using length was even faster than Any(), so this is worth looking at

Reply:
length won't work on ienumerable